bindings/python/PyAPI.cpp

```diff
@@ -77,7 +79,13 @@ PyInit_aud()
 	if(!initializeSequenceEntry())
 		return nullptr;
 
-	if(!initializeSequence())
+	if (!initializeSequence())
```

Comment:
Another consistency comment: let's keep the coding style in audaspace. Can you remove the space of all "if (", "for (", etc. => "if(", "for(" please!